docs/layers/tile-3d-layer.md

```diff
@@ -121,15 +121,6 @@ const layer = new Tile3DLayer({
 })
 ```
 
-##### `DracoLoader` (Object, Optional)
```

Comment:
We should cut this, but we must instead add very clear instructions to the user about how the DracoLoader should be installed.